src/libs/AxelarQueryAPI.ts

```diff
@@ -92,11 +92,13 @@ interface HopParams {
    */
   sourceTokenAddress?: string;
```

Comment:
token id and amount should be provided too, since express depends on that, and they're params to the interchainTransfer method